CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+
+- Added GraphQL API `csvFormattedRawEvents` that returns the values of raw
+  events of the request protocol in csv format String.
```

Comment:
I think it would be worth describing that the delimiter is tab, not comma, such as "the request protocol in csv format string, delimited by tab." Other than this point, I think this PR is good to be merged.